### ISSUE TITLE
Polly Deploy integration

### ISSUE THREAD

**Issue by dazinator** [dazinator/DnnPackager#78]
https://github.com/cantarus/PolyDeploy

DnnPackager v3 will support deployment via:

1. Traditional URL mechanism (i.e < dnn 8)
2. Newer Mechanism that works with locally hosted (IIS) websites only 

But for v3.1 should consider also adding:

3. Via PollyDeploy - supports local and remote dnn websites. (Requires Polly Deploy module and API key to be set up in advance)

**Comment by david-poindexter** [comment]
Absolutely!  I am sitting with the Cantarus folks right now on the bus and have told them of your repo.  You guys should definitely connect!
